routes/api/users.js

```diff
@@ -157,4 +161,51 @@ router.patch(
   }
 );
 
+router.get('/verify/:verificationToken', async (req, res, next) => {
+  try {
+    const { verificationToken } = req.params;
+
+    const user = await User.findOne({ verificationToken });
+    if (!user) return res.status(404).json({ message: 'User not found' });
+
+    await User.findByIdAndUpdate(user._id, {
+      verificationToken: null,
```

Comment:
Czy zapisanie null do verificationToken Ci działa, gdyż w schema masz ustawione że to wymagane pole, sprawdzałaś?

Reply:
Napisałam na slacku :)